content/docs/about/policies/global-attachment.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Global policy attachment
+weight: 20
+---
+
+By default, you must attach policies to resources that are in the same namespace. However, you might have policies that you want to reuse across teams, such as to standardize security protections across your organization.
+
+To do so, you can create policies in a "global" namespace. Then, the policies can attach to resources in any namespace in your cluster through label selectors.
+
+{{< callout type="warning" >}}
+Because it increases the number of policy attachments to calculate, the global policy namespace feature can impact performance at scale. It also changes the standard policy attachment behavior, which can make debugging more difficult. As such, make sure to establish clear guidelines for when and how many global policies to use this feature.
```

Comment:
if im misunderstanding the intent feel free to change
```suggestion
Because it increases the number of policy attachments to calculate, the global policy namespace feature can impact performance at scale. It also changes the standard policy attachment behavior, which can make debugging more difficult. As such, make sure to establish clear guidelines for when and how many global policies can use this feature.
```